utilities/account/fixing_parent/fixingAccountParent.js

```diff
@@ -0,0 +1,96 @@
+// Fixing account parent
+'use strict';
+
+var db = require('../../../server/lib/db'),
+		q  = require('q');
+
+db.initialise();
+
+(function queryAccount() {
+	var accounts;
+	var sqlUpdateMatch = 'UPDATE account a JOIN account b ON a.parent=b.account_number SET a.parent=b.id ;';
+	var sqlAccount 		 = 'SELECT * FROM account;';
+
+	executeUpdate()
+	.then(executeAccount())
+	.then(unmatchAccount)
+	.then(updateParent);
```

Comment:
What about a catch statement?  You should at least let the user know that something went wrong!

(also, you should always end a promise chain with a `.done()` method.